src/auth/basic/SSPI/basic_sspi_auth.cc

```diff
@@ -149,7 +149,7 @@ main(int argc, char **argv)
         /* Clear any current settings */
         username[0] = '\0';
         password[0] = '\0';
-        sscanf(wstr, "%s %s", username, password);  /* Extract parameters */
+        sscanf(wstr, "%255s %255s", username, password);  /* Extract parameters */
 
```

Comment:
The helper should reject a request it cannot parse instead of silently proceeding with truncated field values and/or ignoring trailing garbage after the last field.